server/service/OperationServerService.js

```diff
@@ -92,16 +92,15 @@ exports.getOperationServerOperationName = function (url) {
 exports.putOperationServerLifeCycleState = function (url, body, uuid) {
   return new Promise(async function (resolve, reject) {
     try {
-      let value
       let oldValue = await OperationServerService.getLifeCycleState(uuid)
       let remoteServerLifeCycleState = [];
       let remoteLifeCycleStateEnum = OperationServerService.OperationServerInterfacePac.OperationServerInterfaceConfiguration.lifeCycleStateEnum;
       for (let remoteLifeCycleStateEnumKey in remoteLifeCycleStateEnum) {
-        if (remoteLifeCycleStateEnum[remoteLifeCycleStateEnumKey] == oldValue || remoteLifeCycleStateEnumKey == oldValue) {
-          remoteServerLifeCycleState = [remoteLifeCycleStateEnumKey, remoteLifeCycleStateEnum[remoteLifeCycleStateEnumKey]];
+        if (remoteLifeCycleStateEnumKey == oldValue) {
+          remoteServerLifeCycleState = [remoteLifeCycleStateEnum[remoteLifeCycleStateEnumKey]];
```

Comment:
yes it should be in list

Reply:
How is this (currentState) :
```
let remoteServerLifeCycleState = [];
remoteServerLifeCycleState = [remoteLifeCycleStateEnum[remoteLifeCycleStateEnumKey]];
let value = remoteServerLifeCycleState[0]
if (value !== newValue) {
```
needed instead of this:
```
let remoteServerLifeCycleState;
remoteServerLifeCycleState = remoteLifeCycleStateEnum[remoteLifeCycleStateEnumKey];
if (remoteServerLifeCycleState !== newValue) {
```

Reply:
ok sure